server/src/test/java/org/elasticsearch/indices/breaker/HierarchyCircuitBreakerServiceTests.java

```diff
@@ -723,8 +723,9 @@ public void testTrippedCircuitBreakerDurability() {
 
     public void testAllocationBucketsBreaker() {
         Settings clusterSettings = Settings.builder()
+            // the memory used by the test should be bigger than this value
             .put(HierarchyCircuitBreakerService.TOTAL_CIRCUIT_BREAKER_LIMIT_SETTING.getKey(), "100b")
-            .put(HierarchyCircuitBreakerService.USE_REAL_MEMORY_USAGE_SETTING.getKey(), "false")
+            .put(HierarchyCircuitBreakerService.USE_REAL_MEMORY_USAGE_SETTING.getKey(), "true")
```

Comment:
This looks like an unrelated change, any reason for this?

Reply:
This change is necessary because multiBucketConsumer#accept only checks the  real memory circuit breaker now and needs to be enabled. I changed the test so it is more obvious.